meta/main.yml

```diff
@@ -12,14 +12,6 @@ galaxy_info:
   categories:
   - system
 dependencies:
-  - src: "azavea.java"
```

Comment:
Could we potentially remove these dependencies from here to allow them to be defined at a higher level?

Reply:
I can't remove the version constraints completely, or else Ansible will choose the role defaults, even if `java_version` is defined in a higher level playbook. I compromised by adding variables for `jenkins_java_version` and `jenkins_java_major_version`. When defined in `group_vars` in an encapsulating playbook,  the right version of OpenJDK gets installed.